frontend/src/metabase/common/components/DataPicker/components/DataPickerModal.tsx

```diff
@@ -117,99 +120,127 @@ export const DataPickerModal = ({
     [onChange, onClose, tryLogRecentItem],
   );
 
-  const handleCardChange = useCallback(
```

Comment:
`handleCardChange` has been removed because it was possible to reuse `handleItemSelect` in its place